src/features/defi/providers/fox-farming/api.ts

```diff
@@ -0,0 +1,97 @@
+import { Contract } from '@ethersproject/contracts'
+import { Fetcher, Token } from '@uniswap/sdk'
+import IUniswapV2Pair from '@uniswap/v2-core/build/IUniswapV2Pair.json'
+import dayjs from 'dayjs'
+import { FOX_TOKEN_CONTRACT_ADDRESS, WETH_TOKEN_CONTRACT_ADDRESS } from 'plugins/foxPage/const'
+import { getEthersProvider, makeTotalLpApr, rewardRatePerToken } from 'plugins/foxPage/utils'
```

Comment:
Don't need to change this here but we'll probably want to move this away from the FoxPage domain, will add it to the DeFi improvement issue

Reply:
https://github.com/shapeshift/web/issues/2434